ddtrace/tracer/tracer.go

```diff
@@ -255,28 +248,6 @@ func Start(opts ...StartOption) error {
 	return nil
 }
 
-func storeConfig(c *config) {
```

Comment:
The one call to this function got removed earlier in the v2 process. I want to double check if we still need this, else we can get rid of it.